engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/OpenIdConnectProfile.scala

```diff
@@ -46,13 +46,13 @@ import scala.concurrent.duration.Deadline
   val notBefore: Option[Deadline] = None
 }
 
-object OpenIdConnectUserInfo extends EpochSecondsCodecs with JavaTimeDecoders with JavaTimeEncoders {
+object OpenIdConnectUserInfo extends EpochSecondsCodecs with JavaTimeDecoders with JavaTimeEncoders with EitherCodecs {
   implicit val config: Configuration = Configuration.default
 }
 
 object OpenIdConnectProfile extends OAuth2Profile[OpenIdConnectUserInfo] {
   def getAuthenticatedUser(profile: OpenIdConnectUserInfo, configuration: OAuth2Configuration): AuthenticatedUser = {
-    val userRoles = getUserRoles(profile.email, configuration)
+    val userRoles = getUserRoles(profile.subject.get, configuration)
```

Comment:
Why subject is optional? What will happen when subject will be not defined and we get into this place in code?

Reply:
There is a comment on this in the source code.

Reply:
Indeed, I've missed it